cli/js/web/blob.ts

```diff
@@ -6,11 +6,11 @@ import { ReadableStreamImpl } from "./streams/readable_stream.ts";
 
 export const bytesSymbol = Symbol("bytes");
 
-export function containsOnlyASCII(str: string): boolean {
-  if (typeof str !== "string") {
+export function containsOnlyASCII(s: string): boolean {
```

Comment:
While I agree with many of the changes, a lot of them seem arbitrary and based on your personal preference, is there a code style that we're following that justify these changes?

Reply:
> While I agree with many of the changes, a lot of them seem arbitrary and based on your personal preference, is there a code style that we're following that justify these changes?

In this case, i just wanted parameter name to match other functions in this file